dotnet/samples/Demos/VectorStoreRAG/Program.cs

```diff
@@ -80,8 +80,8 @@
             appConfig.AzureCosmosDBMongoDBConfig.ConnectionString,
             appConfig.AzureCosmosDBMongoDBConfig.DatabaseName);
         break;
-    case "AzureCosmosDBNoSQL":
-        kernelBuilder.Services.AddAzureCosmosDBNoSQLVectorStoreRecordCollection<TextSnippet<string>>(
+    case "CosmosNoSQL":
```

Comment:
Since we use `NoSql` in all other places, should we also drop the uppercase here? And more importantly: are there any other places where we have left `NoSQL`?

Reply:
Good catch, I thought I could all of these.

(.NET naming guidelines forbid upper-case acronyms in general, I think aligning constants like this to the provider name makes a lot of sense).